package.json

```diff
@@ -190,6 +190,24 @@
                 "title": "Clear",
                 "category": "Notebook",
                 "icon": "$(clear-all)"
+            },
+            {
+                "command": "jupyter-kernelManager.viewKernelMessagesAsTree",
+                "title": "Tree View",
+                "category": "Notebook",
+                "icon": "$(list-tree)"
+            },
+            {
+                "command": "jupyter-kernelManager.viewKernelMessagesAsList",
+                "title": "List View",
+                "category": "Notebook",
+                "icon": "$(list-flat)"
+            },
+            {
+                "command": "jupyter-kernelManager.kernelMessageCopy",
```

Comment:
This was very useful when working on IPyWidgets (being able to copy text such as comm_id, msg_id)